LocalSettings.php

```diff
@@ -810,7 +748,103 @@
 		'batmanwiki' => true,
 		'snapwikiwiki' => true,
 	],
-
+	'wgCosmosFooterBackgroundColor' => [
+		'default' => '#c0c0c0',
+	],
+	'wgCosmosRailBlacklistedNamespaces' => [
+		'default' => [
+			'NS_SPECIAL',
```

Comment:
Should without the string, just constants for this and all below. Though we can probably add this to ManageWiki namespaces also.

Reply:
Removed in favor of https://github.com/miraheze/mw-config/pull/4548